### ISSUE TITLE
Binance perpetual - Delay on available balance causing inconsistent orders 

### ISSUE THREAD

**Issue by rapcmia** [hummingbot/hummingbot#6508]
### Describe the bug

Reviewing binance perpetual for couple of hours and observed that the available balance is delayed on updating causing inconsistent order creation. On the screen recording below observed that the strategy is returning of insufficient balance when it should able post both buy and sell order. 

https://github.com/hummingbot/hummingbot/assets/73840223/38d2d901-7d00-46a8-a3fb-d5a337baef2a

- Observed that the available balance is causing inconsistent orders and delay which is also recorded on time `2:13` onwards
- While this is true that there was a a change on the values, still it is not correct since the value of XRP upon testing was around `12 USDT` and both buy and sell order are posted so expected available balance would be around `22 USDT`

### Steps to reproduce

1. Setup Perpetual market making with `binance_perpetual`
2. Set order amount enough to create buy and sell order 
3. Set long refresh time and observe connector behavior 

### Release version

previous versions and dev-1.19.0

### Type of installation

Source

### Attach required files

[logs_perp-binance.log](https://github.com/hummingbot/hummingbot/files/12214914/logs_perp-binance.log)
[perp-binance.yml.log](https://github.com/hummingbot/hummingbot/files/12214924/perp-binance.yml.log) `must remove the .log to be used`

**Comment by nikspz** [comment]
Issue reproduced on dev-1.19.0
![binance perpetual_ issue with delayed available banace update reproduced](https://github.com/hummingbot/hummingbot/assets/83953535/e314fe16-4bc9-478b-9126-27ecade8e31c)

**Comment by leastchaos** [comment]
hi, would this bug be fixed soon? 
it seems that the connector does not update the available balance with the ORDER_TRADE_UPDATE stream data.
![image](https://github.com/hummingbot/hummingbot/assets/64965782/a858e8ca-2f8b-45cf-9fc8-c3752a77f281)
it is only updated via the balance stream which only get updates when there is a change in position and the rest api only.

**Comment by cardosofede** [comment]
@leastchaos how is it going!
just to follow up this issue, if the docs says that account update only sends the update when we have a change on balance, positions or margin type, this means that the real balance will be okey but the available balance not since create and cancel orders are not included in this stream?
would you like to propose a solution for this?
there is a parameter of the connector that uses the information of the inflight orders to do the calculation.
It's defined in the connector base and is called real_time_balance_update

**Comment by leastchaos** [comment]
yes, available balance is only updated in the rest and not the websocket stream.

adding a single line `self.real_time_balance_update = False` to the init of BinancePerpetualDerivative can solve the available balance issue if we are only creating and canceling orders in a single market. e.g. creating orders and canceling orders in BTC/USDT if the strategy is market making on BTC/USDT but if i run on BTC/USDT but create and cancel orders in ETH/USDT, it will not do the update as the connector is not looking for the orders update for other markets. 
so I'm stuck here and did not try to attempt further to  solve this as  I ended up polling more often by changing LONG_POLL_INTERVAL to just 2sec which suited my needs at the time.
